.travis.yml

```diff
@@ -1,95 +1,96 @@
-  - packages/variable-resolver/node_modules
-  - packages/workspace/node_modules
-# end_cache_directories
-  - packages/java-debug/download
-  - packages/debug-nodejs/download
+    # All directories need to be listed here, because Travis does not support globs.
+    # Auto generated by scripts/prepare-travis
+    # start_cache_directories
+    - /tmp/vscode-ripgrep-cache-1.5.7
+    - dev-packages/application-manager/node_modules
+    - dev-packages/application-package/node_modules
+    - dev-packages/electron/node_modules
+    - examples/api-samples/node_modules
+    - examples/browser/node_modules
+    - examples/electron/node_modules
+    - node_modules
+    - packages/callhierarchy/node_modules
+    - packages/console/node_modules
+    - packages/core/node_modules
+    - packages/debug-nodejs/node_modules
+    - packages/debug/node_modules
+    - packages/editor-preview/node_modules
+    - packages/editor/node_modules
+    - packages/editorconfig/node_modules
+    - packages/extension-manager/node_modules
+    - packages/file-search/node_modules
+    - packages/filesystem/node_modules
+    - packages/getting-started/node_modules
+    - packages/git/node_modules
+    - packages/java-debug/node_modules
+    - packages/java/node_modules
+    - packages/json/node_modules
+    - packages/keymaps/node_modules
+    - packages/languages/node_modules
+    - packages/markers/node_modules
+    - packages/merge-conflicts/node_modules
+    - packages/messages/node_modules
+    - packages/metrics/node_modules
+    - packages/mini-browser/node_modules
+    - packages/monaco/node_modules
+    - packages/navigator/node_modules
+    - packages/outline-view/node_modules
+    - packages/output/node_modules
+    - packages/plugin-dev/node_modules
+    - packages/plugin-ext-vscode/node_modules
+    - packages/plugin-ext/node_modules
+    - packages/plugin-metrics/node_modules
+    - packages/plugin/node_modules
+    - packages/preferences/node_modules
+    - packages/preview/node_modules
+    - packages/process/node_modules
+    - packages/python/node_modules
+    - packages/scm/node_modules
+    - packages/search-in-workspace/node_modules
+    - packages/task/node_modules
+    - packages/terminal/node_modules
+    - packages/textmate-grammars/node_modules
+    - packages/tslint/node_modules
+    - packages/typehierarchy/node_modules
+    - packages/typescript/node_modules
+    - packages/userstorage/node_modules
+    - packages/variable-resolver/node_modules
+    - packages/workspace/node_modules
+    # end_cache_directories
+    - packages/java-debug/download
+    - packages/debug-nodejs/download
 before_cache:
   # Runs before the cache is updated, after successful CI
   - rm -f node_modules/@theia/electron/post-install.log
   - rm -rf node_modules/@theia/electron/download
   - rm -rf node_modules/electron
 branches:
   only:
-  - master
+    - master
 env:
   global:
     - CXX=g++-4.8
     - NODE_OPTIONS="--max_old_space_size=4096"
+    - THEIA_TEST_DEBUG=1
```

Comment:
are changes to this file necessary? seems to be a lot of noise and what is `THEIA_TEST_DEBUG`?

Reply:
This is a desperate attempt at understanding why travis fails, this is temporary.

Reply:
ok, please don't forget to revert, i will skip looking at the PR till the build is green to avoid noise from me 😄